pages/common/llm.md

```diff
@@ -11,18 +11,26 @@
 
 `llm "{{Ten fun names for a pet pelican}}"`
 
-- Run a [s]ystem prompt against a file:
+- Run a system prompt against a file:
 
 `cat {{path/to/file.py}} | llm --system "{{Explain this code}}"`
 
 - Install packages from PyPI into the same environment as LLM:
 
 `llm install {{package1 package2 ...}}`
 
-- Download and run a prompt against a [m]odel:
+- Download and run a prompt against a model:
 
 `llm --model {{orca-mini-3b-gguf2-q4_0}} "{{What is the capital of France?}}"`
 
-- Have an interactive chat with a specific [m]odel:
+- Create a system prompt:
```

Comment:
```suggestion
- Create a [s]ystem prompt and save it with a template name:
```

Reply:
Shouldn't you also put square brackets to [t]emplate since it also highlights short options for the example `{{sentient_cheesecakes}}`?

Reply:
Also, I didn't know about using square brackets to highlight short options, and I also couldn't find it on the [contribution guideline](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md). It's probably better to include it in the contribution guideline as well, because there are quite a few commands that don't follow the highlighting rule for short options.

Reply:
> Shouldn't you also put square brackets to [t]emplate since it also highlights short options for the example `{{sentient_cheesecakes}}`?

Yes, updated the suggestion.

> Also, I didn't know about using square brackets to highlight short options, and I also couldn't find it on the [contribution guideline](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md). It's probably better to include it in the contribution guideline as well, because there are quite a few commands that don't follow the highlighting rule for short options.

We have documented this in detail in the style guide at <https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md>, the `CONTRIBUTING.md` file is a quick reference to get started, whereas the style guide (referenced in the CONTRIBUTING file too) contains all the nuances required.

I will try to add a note at the top of the CONTRIBUTING file clarifying this and instead link to the style guide for extended syntax.